python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -312,6 +320,18 @@ def _report_current_usage(
         if self._global_info:
             self._global_info.set_description(resources_status)
 
+    def _update_dataset_metrics(self, op_state):
+        metrics = op_state.op.get_metrics()
+        self._dataset_metrics.inc_bytes_spilled.remote(
+            metrics.get("obj_store_mem_spilled", 0)
```

Comment:
nit: can we use constants/enums to standardize these strings used to reference metrics/stats attributes? we can likely maintain them in `stats.py`.

Reply:
This will be done by Hao's pr